docs/data/sensitive-data/tutorials/sdsi.md

```diff
@@ -0,0 +1,275 @@
+# Submitting jobs from SD Desktop to the HPC environment of CSC
+
+The limited computing capacity of a SD Desktop virtual machines can prevent running heavy analysis tasks
+for sensitive data. This document describes, how heavy computing tasks can be submitted form SD Desktop
+to the Puhti HPC cluster.
+
+Please note following details that limit the usage of this procedure:
+
+   * The service is not yet in full production. Access will be provided by a request for projects that have computing tasks that are compatible with the current status of the service. Contact servicedesk@csc.fi the enable this service.
+   * The *sdsi-client* job submission tool, described in this document, will work only for the approved projects.
+   * Each job reserves always one, and only one, full Puhti node for your task. Try to construct your batch job so that it uses effectively all the 40 computing cores of one Puhti node.
+   * The input files that the job uses must be uploaded to SD Connect before the job submission. Even though the job is submitted from SD Desktop, you can't utilize any files from the SD Desktop VM in the batch job.
+   * The jobs submitted from SD Desktop to Puhti have higher security level that normal Puhti jobs but lower than that of SD Desktop.  
+
+# Security aspects
+
+The increased security of the computing tasks that are submitted with the sdsi-client is based on two main features:
+
+   1. Both input and output data are stored and transported in encrypted format. Encryption is done automatically using the SD Connect methodology.
+   2. For the actual analysis the data is temporarily decrypted to the local disk area of a compute node that is fully reserved for only this one job. Thus there can't be other users in the node during the execution of the job, which effectively eliminates the possibility that other users could access the disk areas, memory or process list of the node during the processing. All the data will be removed from local disk of the node when the job ends.
```

Comment:
Thus there are no other users in the node during the execution of the job, and  other users cannot access the disk areas, memory or process list of the node during the processing.